src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid implements Figure {
```

Comment:
The class IsoscelesTrapezoid should not start with an empty line after the class declaration.